README.md

```diff
@@ -1,60 +1,30 @@
 
+## Packages
+[Lerna in independent mode](https://github.com/lerna/lerna#independent-mode---independent) is used to organize and manage this monorepository and contains utilities each of which are published to NPM. Users can choose one package, some packages, or all of them together.
 
-### General packages
-#### FAST animations
-An animation library that simplifies interactions and animations using the Web Animation API. Provides a convenient abstraction layer over the Web Animation API while still allowing full access to it. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-animation)
-
-#### FAST browser extensions
-An extension for testing real-world localization, theming, and other production type scenarios in web development. Originally devised as a way to visualize prototypes across many different user scenarios. For example, view a website in one of the 194 different localized settings. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-browser-extentions)
-
-#### FAST component contracts
-This package provides the TypeScript typings for all FAST components class-name contracts. These contracts enable strict typing of JSS stylesheets and each component's expectation of which class-names will be made available (and under which keys those class-names reside) to the component at runtime. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-components-class-name-contracts)
-
-#### FAST permutator
-Creates all possible data permutations from a given JSON Schema, allowing for deep testing or display of complex data sets. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-permutator)
-
-#### FAST tslint rules
-Provides [tslint](https://github.com/palantir/tslint) rules for all FAST projects. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-tslint-rules)
-
+[Learn more about FAST-DNA packages on wiki...](https://github.com/Microsoft/fast-dna/wiki/Packages)
 
 ## Policies
 ### Guiding principles, contributions, and governance
-Details including getting started can be found [here](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md).
+Details including getting started can be found [on wiki](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md).
 
 ### Code of conduct
-Details are located [here](https://github.com/Microsoft/fast-dna/blob/master/CODE_OF_CONDUCT.md).
+Details are located [on wiki](https://github.com/Microsoft/fast-dna/blob/master/CODE_OF_CONDUCT.md).
 
 ### Accessibility
-FAST-DNA has partnered with Deque & Microsoft to build accessibility into our components ensuring we are WCAG 2.0 AA compliant. [Readme](https://github.com/Microsoft/fast-dna/wiki/Accessibility)
+FAST-DNA has partnered with Deque & Microsoft to build accessibility into our components ensuring we are WCAG 2.0 AA compliant. [on wiki](https://github.com/Microsoft/fast-dna/wiki/Accessibility)
```

Comment:
"See more in the wiki"

Reply:
I'm trying to be less verbose. I think it's clear with any of these.